packages/neuron-wallet/src/models/dao-utils.ts

```diff
@@ -45,6 +45,10 @@ export default class DaoUtils {
     return DaoUtils.daoScriptInfo
   }
 
+  static cleanInfoWhenSwitchNetwork(): void {
```

Comment:
I think `cleanInfo` is enough for this method

Reply:
updated